less/variables.less

```diff
@@ -24,3 +24,77 @@
 @border-real: #dee2e6;
 
 // Mobile var
+@mobile: ~"(max-width:500px)";
```

Comment:
Good work. Definitely make sure you understand the concept of escaping. You'll see it again in JS.